src/aks-preview/azext_aks_preview/_helpers.py

```diff
@@ -92,3 +95,49 @@ def _trim_fqdn_name_containing_hcp(normalized_fqdn: str) -> str:
     if len(storage_name_without_hcp) > CONST_CONTAINER_NAME_MAX_LENGTH:
         storage_name_without_hcp = storage_name_without_hcp[:CONST_CONTAINER_NAME_MAX_LENGTH]
     return storage_name_without_hcp.rstrip('-')
+
+
+def _fuzzy_match(query, arr):
+    """
+    will compare all elements in @arr against the @query to see if they are similar
+
+    similar implies one is a substring of the other or the two words are 1 change apart
+
+    Ex. bird and bord are similar
+    Ex. bird and birdwaj are similar
+    Ex. bird and bead are not similar
+    """
+    def similar_word(a, b):
+        a_len = len(a)
+        b_len = len(b)
+        if a_len > b_len:  # @a should always be the shorter string
+            return similar_word(b, a)
+        if a == b:
+            return True
+        if a in b:
+            return True
+        if b_len - a_len > 1:
+            return False
+        i = 0
```

Comment:
nit: can be "i, j = 0" for convenience.

Reply:
This doesn't seem to work.

Reply:
I think the syntax is something like i == j = 0
or i,j = 0,0

Reply:
I think that makes it more complicated than it needs to be. They are on separate lines, but in total have the same number of characters for both initializations.

Reply:
@zqingqing1 would you prefer the suggested format or the existing one?

Reply:
I am ok with the current format. Just think it could be shorter.